index.html

```diff
@@ -2554,12 +2553,13 @@ <h3>Using computed attribute values</h3>
           of its descendant <code>&lt;div&gt;</code> elements that do correspond to <a>Script Events</a>
           are different to what they would be if that intermediate <code>&lt;div&gt;</code> element's <code>begin</code>
```

Comment:
Reading this text again, I find the parts about "that element does not correspond to a Script Event" confusing. It shouldn't matter whether or not it is a Script Event. Inheritance applies. Also it gives the impression that under some circumstances, intermediate div can be ignored. 
I would remove the whole sentence starting with "If". Otherwise, I would rephrase saying something like: 
> If a workflow decides to rewrite a DAPT document to remove nested div elements, care should be taken that if those elements had inheritable attributes like begin, the attribute values like begin on the nested divs should be adjusted.

Reply:
I don't like the idea of making this example specifically about rewriting documents - it's about the correct interpretation of a document, regardless of what is being done with it.

Possibly this comment should be raised as a separate issue?

I'd be happy to emphasise further (informatively) that ignoring the intermediate `<div>` is not an acceptable thing to do.

Reply:
My suggestion was a tentative rewrite of what I thought you were trying say. The alternative was to remove the sentence. My main problem is the current emphasis on elements that do not correspond to Script Events. We should not encourage that.

Reply:
Here's another attempt at what I was trying to say, maybe this is less confusing?

>Or, for another example, the computed times of an element in a DAPT document are relative to the begin time of its parent element. If a `<div>` element specifies a `begin` attribute, then the computed times of its child `<div>` elements are relative to that intermediate `<div>` element's begin time, and so on down the hierarchy. It is important to include those intermediate `<div>` elements' times in the computation even if the processing target is an instance of the DAPT data model in which they have no direct equivalent; otherwise the [Script Event](https://www.w3.org/TR/dapt/#dfn-script-event) Begin and End times would be wrong.

Reply:
Change pushed in 33c33fe. On re-reading, I made a couple of small tweaks because I thought the concept of an "intermediate" div wasn't really explained properly.